api/v1/openapi.yaml

```
@@ -2376,6 +2376,12 @@ definitions:
                 - SNAT
                 - DSR
                 - Hybrid
+              dsrMode:
+                type: string
+                enum:
+                  - opt
+                  - ipip
+                  - geneve
```

```suggestion
                  - IP Option/Extension
                  - IPIP
                  - Geneve
```

Let's pretty-print it :)

---

Updated.